Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -289,7 +307,11 @@ def run_checks(
 
           next version_info if version_info.is_a?(Hash) && version_info[:status] && version_info[:messages]
```

Comment:
I wonder if we should modify `version_info` to include resource information as well (though I haven't tested a scenario where this condition is true).

Reply:
I was thinking of including resource information to `version_info` too but I was not sure about the scenario in which this line execute, since the only time 

```ruby
if version_info.is_a?(Hash) && version_info[:status] && version_info[:messages]
```
evaluates to `true` is when `latest_version` method returns a `Hash` from `status_hash` method (check below):

```ruby
...
        if messages.is_a?(Array) && match_version_map.blank?
          puts messages unless json
          next if i + 1 < urls.length

          return status_hash(formula_or_cask, "error", messages, full_name: full_name, verbose: verbose)
        end
...
```

However, I could include resource information to `version_info` if that's what you suggest